ports/stm32/boards/Passport/modules/flows/connect_wallet_flow.py

```diff
@@ -363,8 +367,8 @@ async def export_by_microsd(self):
                 if self.is_skip_address_verification_enabled():
                     if self.is_force_multisig_policy_enabled():
                         result = await InfoPage(
-                            text='For compatibility with {}, Passport will set your Multisig Policy ' +
-                                 'to Skip Verification.'.format(self.sw_wallet['label']))
+                            text='For compatibility with {}, '.format(self.sw_wallet['label']) +
+                                 'Passport will set your Multisig Policy to Skip Verification.').show()
```

Comment:
While we're at it we can remove the `+` operator since it will allocate one more time (`format` already allocates) so we only call `format`.